### PR TITLE
Resolve qgb conflicts with mvp branch

### PR DESCRIPTION
**Pull request by SweeXordious**
Resolves conflicts for MVP branch

### CODE REVIEW

**Comment by SweeXordious** [comment]
Still trying to make the project build with the new added dependencies

File: go.mod [hunk 1]
```diff
@@ -156,6 +145,7 @@ replace (
 	github.com/99designs/keyring => github.com/cosmos/keyring v1.1.7-0.20210622111912-ef00f8ac3d76
 	github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v0.44.1-celestia
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
-	github.com/tendermint/tendermint => github.com/celestiaorg/celestia-core v0.34.14-celestia.0.20220204195923-742fb3a6ab8d
+	github.com/tendermint/tendermint v0.34.13 => github.com/celestiaorg/celestia-core v0.34.14-celestia.0.20220228062426-721562a3544d
+	github.com/tendermint/tendermint v0.35.2 => github.com/celestiaorg/celestia-core v0.34.14-celestia.0.20220228062426-721562a3544d
```
Comment (SweeXordious) [inline go.mod:149]: temp dep

File: go.mod [hunk 2]
```diff
@@ -4,31 +4,29 @@ go 1.17
 
 require (
 	github.com/celestiaorg/nmt v0.8.0
-	github.com/celestiaorg/quantum-gravity-bridge v1.0.4-0.20220301234733-d9cdb31abe27
+	github.com/celestiaorg/quantum-gravity-bridge v1.0.4-0.20220317164417-2f1a818ef0cc
```
Comment (SweeXordious) [inline go.mod:7]: temp dep. Might be replaced by a tag later

**Review by SweeXordious — COMMENTED** [review]
Reminder so I won't forget :D

**Comment by evan-forbes** [comment]
I will take a look at these after one last manual test of the relayer and orchestrator. Thanks for doing this tho :slightly_smiling_face:

**Comment by SweeXordious** [comment]
@evan-forbes do you have some unpushed changes ? so that we're on the same page

**Comment by evan-forbes** [comment]
> do you have some unpushed changes ? so that we're on the same page
not yet, but I will likely have to make more changes after testing

**Comment by evan-forbes** [comment]
whoops, didn't mean to close

**Pull request closed without merging**